MC/config/analysis_testing/json/analyses_config.json

```diff
@@ -31,7 +31,7 @@
             "expected_output": ["AnalysisResults.root"],
             "config": {
                 "mc": "json://${O2DPG_ROOT}/MC/config/analysis_testing/json/analysis-testing-mc.json",
-                "data": "json://${O2DPG_ROOT}/MC/config/analysis_testing/json/analysis-testing-data.json"
+                "data": "json://${O2DPG_ROOT}/MC/config/analysis_testing/json/analysis-testing-data-qa-event-track.json"
```

Comment:
Do you only want to set this for data? Sure, often the MC and data configuration can be used for either. But the idea is to have different configs for data and MC in general.

Reply:
The configs as they are now are ok for both data and MC in terms of analysis. My change only modifies a dependency to spare resources w/o affecting the final results. Usually in MC this is not a big issue given the lower amount of data. I'd go ahead in this way for the moment.